File: trabajos encargados/trabajo encargado semana 1 condicional -- bucles/ejercicios condicionales/ejer7condicional.cpp
#include <iostream>
using namespace std;
int main (){
cout << "ingrese los numeros: " << endl;
int numero7 , numero8;
cin >> numero7 >> numero8;
cout << "ingrese el operador deseado +, -, *, /" << endl;
char operador;
cin >> operador;

switch (operador){
    case '+':
    cout << "la suma es: " << numero7 + numero8 << endl;
        break;

    case '-':
    cout << "la resta es: " << numero7 - numero8 << endl;
        break;

    case '*':
    cout << "la multiplicacion es: " << numero7 * numero8 << endl;
        break;

    case '/':
    if (numero8 != 0)
    cout << "la divicion es: " << numero7 / numero8 << endl;
    else
        cout << "el numero no pued ser cero: " << endl;
        break; 

    default:
        cout << "opercion no valida: " << endl;
}
return 0 ;
}

File: trabajos encargados/trabajo encargado semana 1 condicional -- bucles/ejercicios condicionales/ejer7condicional.py
num = int(input("Ingrese un numero: "))

print("Ingrese los numeros:")
numero7 = int(input())
numero8 = int(input())

operador = input("Ingrese el operador deseado +, -, *, /: ")

if operador == '+':
    print("La suma es:", numero7 + numero8)

elif operador == '-':
    print("La resta es:", numero7 - numero8)

elif operador == '*':
    print("La multiplicacion es:", numero7 * numero8)

elif operador == '/':
    if numero8 != 0:
        print("La division es:", numero7 / numero8)
    else:
        print("El numero no puede ser cero")

else:
    print("Operacion no valida")
